pg/pg.go

```diff
@@ -35,14 +35,19 @@ func NewPG() (*PG, error) {
 	}, nil
 }
 
-func (pg *PG) Migrate(migrationType string) error {
+func (pg *PG) Migrate(migrationType string, migrationPath string) error {
+	if migrationPath == "test" {
+		migrationPath = "file://migrations"
+	} else {
+		migrationPath = "file://pg/migrations"
+	}
```

Comment:
what if instead we remove this part, and just set the migrationPath from where we call Migrate?